### ISSUE TITLE
Cargo.lock not updated/committed for 0.22.0

### ISSUE THREAD

**Issue by ptrcnull** [breard-r/acmed#103]
trying to build with `--locked` reports an error, because the version wasn't bumped in the lockfile:
```
error: the lock file acmed-0.22.0/Cargo.lock needs to be updated but --locked was passed to prevent this
If you want to try to generate the lock file without accessing the network, remove the --locked flag and use --offline instead.
```

**Comment by breard-r** [comment]
Thank you for the report, I completely missed that.

**Comment by breard-r** [comment]
I just released the version 0.22.1, which should fix this.

Also, I updated the script I use to release a new version so it won't happen again.

**Comment by ptrcnull** [comment]
that was fast, thanks!
